src/localplayer.h

```diff
@@ -143,35 +146,36 @@ class LocalPlayer : public Player
 
 	v3f m_position;
 
-	v3s16 m_sneak_node;
+	v3s16 m_sneak_node = v3s16(32767, 32767, 32767);
 	// Stores the max player uplift by m_sneak_node
 	// To support temporary option for old move code
-	f32 m_sneak_node_bb_ymax;
+	f32 m_sneak_node_bb_ymax = 0.0f;
 	// Stores the top bounding box of m_sneak_node
-	aabb3f m_sneak_node_bb_top;
+	aabb3f m_sneak_node_bb_top = aabb3f(0.0f, 0.0f, 0.0f, 0.0f, 0.0f, 0.0f);
 	// Whether the player is allowed to sneak
-	bool m_sneak_node_exists;
+	bool m_sneak_node_exists = false;
 	// Whether recalculation of m_sneak_node and its top bbox is needed
-	bool m_need_to_get_new_sneak_node;
+	bool m_need_to_get_new_sneak_node = true;
 	// Whether a "sneak ladder" structure is detected at the players pos
 	// see detectSneakLadder() in the .cpp for more info (always false if disabled)
-	bool m_sneak_ladder_detected;
+	bool m_sneak_ladder_detected = false;
 	// Whether a 2-node-up ledge is detected at the players pos,
 	// see detectLedge() in the .cpp for more info (always false if disabled).
-	bool m_ledge_detected;
+	bool m_ledge_detected = false;
 
 	// Node below player, used to determine whether it has been removed,
 	// and its old type
-	v3s16 m_old_node_below;
-	std::string m_old_node_below_type;
-	bool m_can_jump;
-	u16 m_breath;
-	f32 m_yaw;
-	f32 m_pitch;
-	bool camera_barely_in_ceiling;
-	aabb3f m_collisionbox;
-
-	GenericCAO *m_cao;
+	v3s16 m_old_node_below = v3s16(32767, 32767, 32767);
+	std::string m_old_node_below_type = "air";
+	bool m_can_jump = false;
+	u16 m_breath = PLAYER_MAX_BREATH;
+	f32 m_yaw = 0.0f;
+	f32 m_pitch = 0.0f;
+	bool camera_barely_in_ceiling = false;
+	aabb3f m_collisionbox = aabb3f(-BS * 0.30f, 0.0f, -BS * 0.30f, BS * 0.30f,
+			BS * 1.75f, BS * 0.30f);
```

Comment:
Could you please add the new line an argument earlier, so the X,Y,Z values are grouped nicely